docs/source/builtin-plugins.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Apollo Server plugins
+sidebar_title: Overview
+---
+
+**Plugins** extend Apollo Server's core functionality by performing custom operations in response to certain events. These events correspond to individual phases of the GraphQL request lifecycle, and to the lifecycle of Apollo Server itself.
+
+Certain Apollo Server features are distributed as plugins in the `apollo-server` library, some of which are installed automatically in certain cases. You can also [create custom plugins](./integrations/plugins/).
+
+## List of native plugins
+
+| Name | Description |
+|------|-------------|
+| [Usage reporting](./api/plugin/usage-reporting/) | Gathers helpful operation usage data and reports it to [Apollo Studio](https://www.apollographql.com/docs/studio/) for visualization, alerting, and more. |
+| [Schema reporting](./api/plugin/schema-reporting/) | Automatically reports the server's schema to [Apollo Studio](https://www.apollographql.com/docs/studio/) on startup to enable schema history and up-to-date metrics. |
+| [Inline trace](./api/plugin/inline-trace/) | Used primarily by [federated subgraphs](https://www.apollographql.com/docs/federation/) to include operation trace data in responses to the gateway. | 
```

Comment:
want to add empty articles for cache control and landing page plugins in this PR? though I'm not positive if we want landing page to be one page, two pages (default / playground), or three (localdefault, productiondefault, playground).  probably two.